scripts/dali-dev-scale-export.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+PARA_PORT=9188
```

Comment:
please use port we use in other places, please try to make scripts more consistent regarding defaults

Reply:
fixed.